snippets/page.json

```diff
@@ -52,6 +52,16 @@
         ],
         "description": "Snippet: Page Field"
     },
+    "Snippet: Part": {
+        "prefix": "tpart",
+        "body": [
+            "part(${1:MyPart}; ${2:PartSource})",
+            "{",
+            "\t${0}",
+            "}"
+        ],
+        "description": "Snippet: Page Field"
```

Comment:
I think the "description" property should be changed accordingly 😉 .

Reply:
hi @rvanbekkum 
Thanks for the suggestion, I have changed the description. If you find any suggestions let me know about it.